lib/pf/node.pm

```diff
@@ -173,8 +183,9 @@ sub node_delete {
 
     require pf::locationlog;
     # TODO that limitation is arbitrary at best, we need to resolve that.
-    if ( defined( pf::locationlog::locationlog_view_open_mac($mac) ) ) {
-        $logger->warn("$mac has an open locationlog entry. Node deletion prohibited");
+    my ($results, $msg) = _can_delete($mac);
+    if (!$results) {
+        $logger->warn($msg);
```

Comment:
1. Is TODO comment still necessary ?
1. I will prefer `$result` in place `$results` like in other files.

Reply:
@jrouzierinverse, what about the TODO item ?

Reply:
I find $can_delete is more clear on the value is for.